src/NHibernate/Impl/AbstractSessionImpl.cs

```diff
@@ -380,31 +424,31 @@ public virtual ISQLQuery CreateSQLQuery(string sql)
 
 		protected internal virtual IQueryExpressionPlan GetHQLQueryPlan(IQueryExpression queryExpression, bool shallow)
 		{
-			using (new SessionIdLoggingContext(SessionId))
+			using (BeginContext())
 			{
 				return _factory.QueryPlanCache.GetHQLQueryPlan(queryExpression, shallow, EnabledFilters);
 			}
 		}
 
 		protected internal virtual NativeSQLQueryPlan GetNativeSQLQueryPlan(NativeSQLQuerySpecification spec)
 		{
-			using (new SessionIdLoggingContext(SessionId))
+			using (BeginContext())
 			{
 				return _factory.QueryPlanCache.GetNativeSQLQueryPlan(spec);
```

Comment:
Similar to previous case, but actually never triggering a check status, so leaving it as is.